src/providers/google-vertex-ai/imageGenerate.ts

```diff
@@ -0,0 +1,77 @@
+import { GOOGLE_VERTEX_AI } from '../../globals';
+import { EmbedParams } from '../../types/embedRequestBody';
+import { ErrorResponse, ImageGenerateResponse, ProviderConfig } from '../types';
+import { generateInvalidProviderResponseError } from '../utils';
+import { GoogleErrorResponse } from './types';
+import { GoogleErrorResponseTransform } from './utils';
+
+interface GoogleImageGenParams extends EmbedParams {}
+
+interface GoogleImageGenInstanceData {
+  prompt: string;
+}
+
+export const GoogleImageGenConfig: ProviderConfig = {
+  input: {
```

Comment:
This should be `prompt` instead of `input` because OpenAI schema accepts `prompt`
https://platform.openai.com/docs/api-reference/images/create#images-create-prompt